### ISSUE TITLE
Date of Birth cannot be updated 

### ISSUE THREAD

**Issue by mahmoud-adel-mahmoud** [SEGUC17/Foobar#26]
1: Severity: ( low )

2:Reported: by Mahmoud

3:Description: When signing up and NOT providing Date of birth, in my profile page, I can't edit my date of birth although i did not specify it when signing up.

4: Steps to reproduce the issue: Sign up. Don't provide Date of Birth. Log in with the new account. Navigate to edit profile page. Try to edit the date of birth.

5: Expected result: I should be able to edit my date of birth, although i did not specify it in the sign up phase.

**Comment by mahmoud-adel-mahmoud** [comment]
<img width="527" alt="screen shot 2017-05-01 at 2 52 35 pm" src="https://cloud.githubusercontent.com/assets/25283016/25580070/d89f7fb4-2e7d-11e7-9d8d-2c3b87dd4c0a.png">

**Comment by markarsanious** [comment]
Yeah that's perfectly intended. Not all the info can be updated. Only the ones that won't affect your account. Email and Date of birth can't be modified cuz they are identifying features for your account.
just like on facebook you can't modify some features like email for example if it is ur primary one

**Comment by mahmoud-adel-mahmoud** [comment]
The problem is not when i'm editing after i've inserted it in the sign up page. It is if I didn't insert it and left it without any entry although it was written that it is required. So, logically If I didn't enter the birthdate from the beginning, I should have the opportunity to enter it at least once after signing up

**Comment by markarsanious** [comment]
you are talking now about the editing part so I'm telling u it is intended not to edit it
